### DISCUSSION TITLE
Can I open Programs with wine cmd?

### DISCUSSION THREAD

**Discussion by ghost** [bottlesdevs/Bottles#165] [Q&A]
I want to execute some batch scripts with wine cmd in bottles,so does that work?

**Comment by mirkobrombin** [comment]
Just open the CMD and execute the script. Note that the CMD doesn't work on the Flatpak at the moment

**Reply by ghost** [reply]
But I mean that I can run a executable directly with winecmd instead of opening winecmd and then launching the script there.

**Reply by mirkobrombin** [reply]
I'm bit confused

**Reply by ghost** [reply]
I mean something like right clicking the .bat file in bottles and then selecting "open with winecmd".

**Reply by mirkobrombin** [reply]
> I mean something like right clicking the .bat file in bottles and then selecting "open with winecmd".

Technically it already works like this. When you choose a .bat file, it is started by the CMD. If what you want is to see the CMD while the file is running, this feature is currently not supported.

**Reply by ghost** [reply]
Can you add it in the future,that I can see the output?
